### DISCUSSION TITLE
How do I add Github Statistics to my readme

### DISCUSSION THREAD

**Discussion by RDE3** [community/community#55412] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I wan to display statistics in my profile

**Comment by Devanshu-17** [comment]
Hey, Follow these steps:

1. Create a new repository with the repo name as your username.
2. Add a readme file in the repo.
3. Paste below code and change the username in the code with your name:

```bash
<p align="left"> <img src="https://komarev.com/ghpvc/?username=Devanshu-17&label=Profile%20views&color=brightgreen&style=flat-square" alt="Devanshu-17" /> </p>
```

**Comment by queenofcorgis** [comment]
@RDE3 @Devanshu-17 GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
